db/migrate/20210819145618_update_network_router_admin_state_up.rb

```diff
@@ -0,0 +1,20 @@
+class UpdateNetworkRouterAdminStateUp < ActiveRecord::Migration[6.0]
+  class NetworkRouter < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    change_column :network_routers, :admin_state_up,
+      "boolean USING COALESCE(admin_state_up::BOOLEAN, FALSE)"
```

Comment:
this generates
```
ALTER COLUMN "admin_state_up" TYPE boolean USING COALESCE(admin_state_up::BOOLEAN)
```

since the column is currently nullable (and we want to remove nulls) we are removing nulls along the way using `COALESCE`

going from string to boolean requires an explicit cast anyway (`USING admin_state::BOOLEAN`), so it doesn't add too much complexity

Reply:
updated `boolean` to `BOOLEAN`